Build: Module not found: Error: Can't resolve 'child_process'

**What version of textlint are you using?**

textlint 8.0.0

**What did you do? Please include the actual source code causing the issue.**

build textlint by webpack and met following error:

```
ERROR in ./~/shelljs/src/exec.js
Module not found: Error: Can't resolve 'child_process' in '/Users/azu/.ghq/github.com/textlint/textlint.github.io/node_modules/shelljs/src'
 @ ./~/shelljs/src/exec.js 6:12-36
 @ ./~/shelljs/src ^\.\/.*$
 @ ./~/shelljs/shell.js
 @ ./~/textlint/lib/util/find-util.js
 @ ./~/textlint/lib/engine/textlint-engine-core.js
 @ ./~/textlint/lib/textlint-engine.js
 @ ./~/textlint/lib/index.js
 @ ./src/components/TextlintDemo/TextlintEditor.js
 @ ./src/components/TextlintDemo.js
 @ ./src/components/App.js
 @ ./src/index.js
```

**What did you expect to happen?**

Pass and can use textlint on browser.

**What actually happened? Please include the actual, raw output from textlint.**

Error

--- a/packages/textlint/package.json
+++ b/packages/textlint/package.json
@@ -59,11 +59,13 @@
     "mocha": "^3.0.2",
     "npm-run-all": "^4.0.2",
     "power-assert": "^1.3.1",
+    "shelljs": "^0.7.7",
     "textlint-plugin-html": "^0.1.2",
     "textlint-rule-helper": "^2.0.0",
     "textlint-rule-no-todo": "^2.0.0",
     "textlint-rule-preset-ja-spacing": "^2.0.0",
-    "textlint-rule-preset-jtf-style": "^2.0.0"
+    "textlint-rule-preset-jtf-style": "^2.0.0",
+    "webpack": "^2.5.1"
   },
   "dependencies": {
     "bluebird": "^3.0.5",
@@ -83,10 +85,9 @@
     "mkdirp": "^0.5.0",
     "object-assign": "^4.0.1",
     "optionator": "^0.8.0",
-    "path-to-glob-pattern": "^1.0.1",
+    "path-to-glob-pattern": "^1.0.2",
     "rc-config-loader": "^1.0.2",
     "read-pkg": "^1.1.0",
-    "shelljs": "^0.7.6",
     "string-width": "^1.0.1",
     "structured-source": "^3.0.2",
     "text-table": "^0.2.0",

--- a/packages/textlint/src/util/find-util.js
+++ b/packages/textlint/src/util/find-util.js
@@ -4,7 +4,13 @@ const pathToGlob = require("path-to-glob-pattern");
 const glob = require("glob");
 const path = require("path");
 const fs = require("fs");
-const shell = require("shelljs");
+const isFile = (filePath) => {
+    try {
+        return fs.statSync(filePath).isFile();
+    } catch (error) {
+        return false;
+    }
+};
 /**
  * filter files by config
  * @param {string[]} patterns glob patterns
@@ -33,7 +39,7 @@ export function findFiles(patterns, options = {}) {
     };
     patterns.forEach(pattern => {
         const file = path.resolve(cwd, pattern);
-        if (shell.test("-f", file)) {
+        if (isFile(file)) {
             addFile(fs.realpathSync(file));
         } else {
             glob.sync(pattern, {

